3711/3/views/places/new.erb

```diff
@@ -0,0 +1,69 @@
+<style>
```

Comment:
it would be better if you create couple `.css` files and move this styles from here